express/blocks/ratings/ratings.js

```diff
@@ -331,6 +332,9 @@ export default function decorate($block) {
       const $votes = createTag('span', { class: 'rating-votes' });
       $votes.innerHTML = `<strong>${rating} / 5</strong> - ${ratingAmount} ${votesText}`;
       $stars.appendChild($votes);
+      if (rating > 4.2) {
```

Comment:
I think we should make it dependent on the block option (`show-totals`) rather than a hardcoded value.